src/main/java/controller/BlackJackGame.java

```diff
@@ -0,0 +1,61 @@
+package controller;
+
+import common.GamerDto;
+import domain.PlayerResult;
+import domain.card.Deck;
+import domain.gamer.Dealer;
+import domain.gamer.Player;
+import view.InputView;
+import view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class BlackJackGame {
+    public void play() {
+        Deck deck = Deck.create();
+        List<String> playerNames = InputView.inputPlayerNames();
+        List<Player> players = playerNames.stream()
+                .map(name -> new Player(deck.getInitCards(), name))
+                .collect(Collectors.toList());
+        Dealer dealer = new Dealer(deck.getInitCards());
+
+        OutputView.printInitGamersState(GamerDto.of(dealer), players.stream().map(GamerDto::of).collect(Collectors.toList()));
```

Comment:
해당 코드는 가독성이 매우 떨어지는데요.
그 이유는 `stream`을 한 줄로 사용하였고, 매개변수로 넘기는 동시에 로직을 처리하여 그렇다고 보여집니다.

```suggestion
        OutputView.printInitGamersState(gamersState);
```

이러한 형태로 만들 수 있다면 가독성이 상승할 것 같아요!

Reply:
넵 그렇게 수정했습니다!